engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/sink/KafkaAvroSinkFactoryWithEditor.scala

```diff
@@ -85,12 +85,11 @@ class KafkaAvroSinkFactoryWithEditor(val schemaRegistryProvider: SchemaRegistryP
     val schemaData = schemaDeterminer.determineSchemaUsedInTyping.valueOr(SchemaDeterminerErrorHandler.handleSchemaRegistryErrorAndThrowException)
     val schemaUsedInRuntime = schemaDeterminer.toRuntimeSchema(schemaData)
 
-    val validationMode = extractValidationMode(params(SinkValidationModeParameterName).asInstanceOf[String])
-    val sinkValue = AvroSinkValue.applyUnsafe(sinkValueParameter.get, parameterValues = params)
+    val sinkValue = AvroSinkValue.applyUnsafe(state.sinkValueParameter, parameterValues = params)
     val key = params(SinkKeyParamName).asInstanceOf[LazyParameter[CharSequence]]
 
     new KafkaAvroSink(preparedTopic, versionOption, key, sinkValue, kafkaConfig, schemaRegistryProvider.serializationSchemaFactory,
-      schemaData.serializableSchema, schemaUsedInRuntime.map(_.serializableSchema), clientId, validationMode)
+      schemaData.serializableSchema, schemaUsedInRuntime.map(_.serializableSchema), clientId, ValidationMode.strict)
```

Comment:
I wonder, does it have any effect/causes problem when "Latest version" is chosen? E.g. when we want to handle compatible changes?

Reply:
I don't see issue with it. The idea is that our schema evolution mechanism handle this situation. From user perspective he/she should see definition of latest version. I see other problem - with handling defaults but will left comment in the place where it exists.